src/components/Footer/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+export default class Footer extends Component {
+  render() {
+    const Footer = styled.form`
```

Comment:
why form 🤔 ?
Recommend checking http://flexboxfroggy.com/, it's a nice layout model